src/common/asb/Asb.c

```diff
@@ -2899,8 +2889,8 @@ static int RemediateEnsureAutomountingDisabled(char* value, void* log)
 static int RemediateEnsureKernelCompiledFromApprovedSources(char* value, void* log)
 {
     UNUSED(value);
-    UNUSED(log);
-    return 0; //TODO: add remediation respecting all existing patterns
+    OsConfigLogInfo(log, "Automatic remediation is not possible");
```

Comment:
This isn't an error? OsConfigLogError?

Reply:
Not an error. It's a FYI. Audit is separate.